.storybook/vitest.setup.ts

```
@@ -1,6 +1,6 @@
 import { beforeAll } from "vitest";
 import { setProjectAnnotations } from "@storybook/nextjs-vite";
-// biome-ignore lint/style/noNamespaceImport: <explanation>
+// biome-ignore lint/performance/noNamespaceImport: ignore
```

Biome (1.9.4)

[error] failed to parse category "lint/performance/noNamespaceImport"



(suppressions/parse)